File: common/src/main/java/dev/mayaqq/estrogen/registry/sounds/MothFlyingSoundInstance.java
package dev.mayaqq.estrogen.registry.sounds;

import dev.mayaqq.estrogen.registry.EstrogenSounds;
import dev.mayaqq.estrogen.registry.entities.MothEntity;
import net.minecraft.sounds.SoundSource;

public class MothFlyingSoundInstance extends MothSoundInstance {

    public MothFlyingSoundInstance(MothEntity moth) {
        super(moth, EstrogenSounds.MOTH_LOOP.get(), SoundSource.NEUTRAL);
    }
}

File: src/main/kotlin/dev/mayaqq/estrogen/client/content/sounds/MothFlyingSoundInstance.kt
package dev.mayaqq.estrogen.client.content.sounds

import dev.mayaqq.estrogen.content.EstrogenSounds
import dev.mayaqq.estrogen.content.entities.MothEntity
import net.minecraft.sounds.SoundSource

class MothFlyingSoundInstance(moth: MothEntity) : MothSoundInstance(moth, EstrogenSounds.MOTH_LOOP.value!!, SoundSource.NEUTRAL)